### ISSUE TITLE
Needs read only view

### ISSUE THREAD

**Issue by NightJar** [silverstripe/silverstripe-versioned-admin#120]
There is currently no "read only" version of the `HistoryViewerField`

This means that e.g. when attempting to apply the history viewer to a versioned DataObject, one can only see history when they are `edit`ing the object.

When accessing the object from the `view` state of `GridFieldDetailForm` `itemRequest`, a simple read only field renders, which does not match the entwine selector to boot the react field.

**Comment by robbieaverill** [comment]
Does something like this help?
```php
public function performReadonlyTransformation()
{
    return clone $this; // doesn't need a different readonly state for this app
}
```

I'd probably argue that this is a bug

**Comment by NightJar** [comment]
>I'd probably argue that this is a bug

I was wondering about that, but I went with enhancement due to it currently not having a read only view, and it would after the issue is closed.

**Comment by robbieaverill** [comment]
Right, I guess I'm thinking of it more as "it doesn't work in a readonly context" e.g. when viewing a record rather than editing it.

**Comment by NightJar** [comment]
Yeah, it's a fine line. It doesn't work in that one does not see the history viewer as expected, however it does work in that it does not cease operation. One simply sees a generic ReadOnlyField with no content.
